articles/sso/index.md

```diff
@@ -0,0 +1,48 @@
+---
+url: /sso
+description: Introduction to Single Sign On (SSO) with Auth0.
+---
+
+# What is SSO (Single Sign On)?
+
+SSO (Single Sign On) is when a user logs in to one application and then is "automatically" signed in to other applications, regardless of platform, technology and domain.
+
+An example of SSO is how Google implements login for their products: Gmail, YouTube, Analytics, etc. When you first login to Gmail and then go to another Google service such as YouTube you won't be prompted for credentials again.
+
+The way this works is by means of a "central service" (with Google this is https://accounts.google.com). When you first login a cookie gets created on this central service. Then when you try to access the second application, you get redirected to the central service, but since you already have a cookie, you get redirected to the app directly with a token, which means you're already logged in.
+
+## How to implement SSO with Auth0
+
+To enable SSO for one of your client applications(each client is independent) go to the [Clients section of the dashboard](${uiURL}/#/applications). Click on the **Settings** gear icon of the client you wish to add SSO.
+
+Then scroll down to toggle "Use Auth0 instead of the IdP to do Single Sign On".
+
+![](/media/articles/sso/single-sign-on/sso-checkbox.png)
+
+::: panel-info Note
+For SSO to work you must setup the Identity Provider(s) you wish to use. This involves getting a Client Key and Client Secret from the IdP. Click [here](/identityproviders) to learn how to setup various providers.
+:::
+
+This turns the SSO flag is on, this flag can also be set using the API. When this flag is set Auth0 will maintain an SSO session for the user.  This session will last a maximum of 7 days if a user remains active, but will terminate after 3 days of inactivity if the user does not remain active.  To be considered active, a user must access an application registered in Auth0 (the same account as the application that created the session).  
```

Comment:
@angelaLazar Since we no longer use "Applications", but "Clients" instead, maybe this needs to change:
"To be considered active, a user must access an application registered in Auth0 (the same account as the application that created the session)." -> "To be considered active, a user must access a Client registered in Auth0 (the same account as the Client that created the session)."